### DISCUSSION TITLE
function from index.js not running when called for in an electron window..

### DISCUSSION THREAD

**Discussion by 2emk7** [community/community#45481] [Projects and Issues]
before i introduce my issue i want to give a little rundown on the project. It gets a list of names from a file (latest.log) and filters out the usernames and looks up the stats for them and sends the stats to an electron app that displays the stats via a table.

PROBLEM - For the search bar on the electron app, i want to be able to look up players and return the stats of the player in the table (just like before). I thought that using pretty much the same code as before would work (with a little tweaking) and it didn't. It seems to run fine as i get no error alert but the playerstats wont show on the table.

If you want to run the whole projects, just replace filePath with the path to your minecraft .log file. You do not need to do anything if you want to test out the searh bar.

code for project:<a href="https://github.com/2emk7/chan-bedwars-overlay">GitHub - 2emk7.</a>[<a href="https://github.com/2emk7/chan-bedwars-overlay" target="_blank" title="New Window">^</a>]

Ive tried multiple versions of the code for the getPlayerStats function but they all don't work. I dont get an error code either when i run it. I also tried moving the code to the html doc but i need the two following in the same document for it to work:

<pre>const hypixel = new Hypixel.Client('77684239-5a07-4ea4-bc9c-2f07db9fddb7');
const player = await hypixel.getPlayer(inputValue);
</pre> 
and i get an error when i try to put the definition of hypixel in the html document.

**Comment by jge162** [comment]
@2emk7 Without being able to see the full code, it's difficult to know for sure what the issue is. Nonetheless, one thing you could try is to add console.log statements in the getPlayerStats function to see if the data is being retrieved correctly and if there are any errors being thrown. You could also try adding a console.log statement after the const player = await hypixel.getPlayer(inputValue); line to see if the player data is being returned correctly.

it may be helpful to seek help from the community of the Hypixel API library or the Electron community for more specific advice.

**Reply by 2emk7** [reply]
I have a link to the code in my post.
the only error i am currently getting is 
(node:13588) UnhandledPromiseRejectionWarning: Error: An object could not be cloned.
    at t.ipcRendererInternal.send (node:electron/js2c/renderer_init:2:11558)
    at EventEmitter.<anonymous> (node:electron/js2c/renderer_init:2:11030)

I am currently trying to find out why this error happens and how to fix it, but so far i have no luck.

**Reply by jge162** [reply]
issue with cloning an object in the Electron renderer process. Sorry can’t help. Wrong forum. Anything you need GitHub related?  Tia
